examples/RapidXML/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+
+# Copyright (c) 2016, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("${CMAKE_CURRENT_SOURCE_DIR}/../common.cmake")
+
+project(download-RapidXML)
+
+# download RapidXML
+hunter_add_package(RapidXML)
+
+# RapidXML available
+find_package(RapidXML REQUIRED CONFIG)
+
+add_executable(foo foo.cpp)
```

Comment:
The name of the target was my guess. I think the real name is `RapidXML::rapidxml`. To avoid such errors you can test changes with [jenkins.py](https://github.com/ruslo/hunter/wiki/dev.hunter.root#jenkinspy) script.